lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -512,13 +512,13 @@ def compile_method(symbol_name, constant, method)
             when :req
               name
             when :opt
-              "#{name} = _"
+              "#{name} = T.unsafe(nil)"
```

Comment:
This will only work for gems RBIs.

Do we need to apply the same patch to DSLs?

Reply:
I assume if we are generating "_" for DSLs , It needs to be changed too so sorbet doesn't complain.

Reply:
I added a commit to fix it too.